### PR TITLE
Coding practice 24

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

**Review by anushreeSonak — APPROVED** [review]

File: simple/src/codingPractice/SpecificWord.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        System.out.println("Is the color " + searchColor + " present in the list? " + flag);
+
+        // Specify the word to search for
+        String searchColorTwo = "White";
+        Predicate<String> containWordOne = word -> word.equals(searchColorTwo);
+        boolean flagOne = colors.stream().anyMatch(containWordOne);
+        // Print the result
+        System.out.println("\nIs the word " + searchColorTwo + " present in the list? " + flagOne);
+    }
+}
```
Comment (prashantkawa) [inline simple/src/codingPractice/SpecificWord.java:29]: Try writing logic in separate method

File: simple/src/codingPractice/SpecificWord.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        // Check if the list contains the specified color using a lambda expression
+        Predicate<String> containWord = word -> word.equals(searchColor);
+        boolean flag = colors.stream().anyMatch(containWord);
+        System.out.println("Is the color " + searchColor + " present in the list? " + flag);
+
+        // Specify the word to search for
+        String searchColorTwo = "White";
+        Predicate<String> containWordOne = word -> word.equals(searchColorTwo);
+        boolean flagOne = colors.stream().anyMatch(containWordOne);
```
Comment (prashantkawa) [inline simple/src/codingPractice/SpecificWord.java:25]: repeating same steps again and again

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/SpecificWord.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SpecificWord.java]: Done

File: simple/src/codingPractice/SpecificWord.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SpecificWord.java]: Done

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**